code/system/application/perception/src/detectlane/detectlane.cpp

```diff
@@ -249,8 +266,7 @@ void DetectLane::nextContainer(odcore::data::Container &c)
           m_lines.col(0)(m_midRegion),m_lines.col(1)(m_midRegion),
           (m_minRow));
       // Factor 4 is assumed length between minRow and Maxrow in real life
-      double theta = atan((d2-d1) / (double)4);
-
+      double theta = atan((d2-d1) / (double)15);
```

Comment:
Instead of all casting, you can simply write:

   ... / 15.0);

and let the compiler do the job for us.